analysis_notebooks/Geography Analysis.ipynb

```diff
@@ -405,17 +410,13 @@
     "        print(country_count_phrases[-1-i])\n",
     "   \n",
     "def subregion_counts_num():\n",
-    "    if is_country_counts:\n",
-    "        print(\"Subregion Counts\\n\")\n",
-    "        total_subregion = sum(subregion_counts.values())\n",
-    "        for subregion, count in sorted(subregion_counts.items(), key=lambda x: x[1], reverse=True):\n",
-    "            print(\"{0}: {1}    {2}%\".format(subregion, count, round(100.*count/total_subregion)))\n",
-    "    else:\n",
-    "        print(\"Subregion counts for gps labels coming soon\")\n",
-    "\n",
+    "    print(\"Subregion Counts\\n\")\n",
```

Comment:
can we include a check to only display subregion analyses if the dataset contains at least two distinct subregions

Reply:
Done, line 163 in geography_based.py 

```
 # only create mapping if there are at least 2 unique subregions
    if len(subregion_to_id_map.keys()) >= 2:
```